lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,146 @@
+# (c) 2017, Jason Vanderhoof <jason.vanderhoof@cyberark.com>, Oren Ben Meir <oren.benmeir@cyberark.com>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: conjur_variable
+    version_added: "2.5"
+    short_description: Fetch credentials from CyberArk Conjur.
+    description:
+      - Retrieves credentials from Conjur using the server's Conjur identity.
+    requirements:
+      - The machine running Ansible has been given a Conjur identity.
+    options:
+      _term:
+        description: Variable path
+        required: True
+"""
+
+EXAMPLES = """
+  - debug
+    msg: {{ lookup('conjur_variable', '/path/to/secret') }}
+"""
+
+RETURN = """
+  _raw:
+    description:
+      - Value stored in Conjur.
+"""
+
+import os.path
+from ansible.errors import AnsibleError, AnsibleParserError
+from ansible.plugins.lookup import LookupBase
+from base64 import b64encode
+from netrc import netrc
+from os import environ
+from time import time
+from urllib import quote_plus
```

Comment:
Improve Python 3 compatibility: use [`six`](http://docs.ansible.com/ansible/devel/dev_guide/developing_python3.html#bundled-six) for `quote_plus`.